digdag-storage-s3/src/main/java/io/digdag/storage/s3/S3StorageFactory.java

```diff
@@ -30,6 +31,13 @@ public Storage newStorage(Config config)
             client.setEndpoint(config.get("endpoint", String.class));
         }
 
+        if (config.has("path-style-access")) {
+            client.setS3ClientOptions(
+              new S3ClientOptions().withPathStyleAccess(
+                config.get("path-style-access", Boolean.class)
```

Comment:
this should be config.get("path-style-access", boolean.class, false)

Reply:
`withPathStyleAccess` is a deprecated method. According to documents, here should use S3ClientOptions.Builder.